plugin/trino-mongodb/src/main/java/io/trino/plugin/mongodb/MongoSession.java

```diff
@@ -433,6 +433,11 @@ private static Optional<Document> buildPredicate(MongoColumnHandle column, Domai
             return Optional.of(documentOf(name, isNotNullPredicate()));
         }
 
+        // Disable pushdown on json columns, see https://github.com/trinodb/trino/issues/13536
+        if (TypeUtils.isJsonType(type)) {
```

Comment:
Static import `isJsonType`. The method name is descriptive without the class name.

Reply:
Yeah, this could be simplified